.eslintrc.js

```diff
@@ -38,10 +37,15 @@ module.exports = {
         'import/no-internal-modules': 'off',
         'no-unused-expressions': 'off'
       }
+    },
+    {
+      files: ['**/index.ts'],
+      rules: {
+        'jsdoc/require-description-complete-sentence': 'off'
```

Comment:
This is off but changed in a lot of comments below. Is that intended?

Reply:
Yes, this was intended. The rule `require-description-complete-sentence` adds a `.` even in `index.ts` files which was not needed. The rule itself has no way of adding exceptions for specific cases, so I had to turn it off this way.